stripper/ze_ffxii_feywood_b6_3k.cfg

```diff
@@ -0,0 +1,205 @@
+;logic_auto changes
+modify:
+{
+	match:
+	{
+		"classname" "logic_auto"
+		"hammerid" "1153"
+	}
+	insert:
+	{
+		"OnNewGame" "weapon_meleeKill20-1"
+		"OnNewGame" "weapon_taggrenadeKill21-1"
```

Comment:
Missed this originally, these surely aren't necessary? They don't exist in the map.